mlflow/store/db/utils.py

```diff
@@ -100,6 +100,13 @@ def make_managed_session():
         except MlflowException:
             session.rollback()
             raise
+        except sqlalchemy.exc.OperationalError as e:
+            session.rollback()
+            _logger.warning(
```

Comment:
Since we're raising this exception, can the logger level reflect that?
This will automatically add the stack trace as an ERROR log event so that the caught exception doesn't need to be string cast within the message statement. 

See: https://docs.python.org/3.8/library/logging.html#logging.Logger.exception

```suggestion
            _logger.exception(
```